cmd/ops_agent_uap_plugin/service_linux.go

```diff
@@ -82,21 +108,22 @@ func (ps *OpsAgentPluginServer) Start(ctx context.Context, msg *pb.StartRequest)
 	}
 
 	// Ops Agent config validation
-	if err := validateOpsAgentConfig(pContext, ps.runCommand, pluginStateDir); err != nil {
+	if err := validateOpsAgentConfig(pContext, pluginInstallDir, ps.runCommand); err != nil {
```

Comment:
Is there a particular reason why we changed to use the Install directory instead of the state directory for the config now?

Reply:
So the plugin install directory is different from the plugin State directory. We will have to run the conf generator binary to verify the provided Ops Agent config. The conf generator locates under the `pluginInstallDir` not the `pluginStateDir`.